src/test/java/seedu/address/model/pet/PetCertificateTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.pet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class PetCertificateTest {
+
+    @Test
+    public void constructor_null_emptyString() {
+        PetCertificate pc = new PetCertificate(null);
+        PetCertificate expected = new PetCertificate("");
+        assertEquals(pc, expected);
+    }
+
+    @Test
+    public void getCertificate() {
+        PetCertificate pc = new PetCertificate("AVA");
+        assertEquals(pc.getCertificate(), "AVA");
+    }
+
+    @Test
+    public void isSamePetCertificate() {
+        PetCertificate pc = new PetCertificate("AVA");
+        assertEquals(pc, pc);
+    }
+
+    @Test
+    public void equals() {
+        PetCertificate pc1 = new PetCertificate("AVA");
+        PetCertificate pc2 = new PetCertificate("AVA");
+        assertEquals(pc1, pc2);
+    }
+
+    @Test
+    public void notEquals() {
+        PetCertificate pc1 = new PetCertificate("AVA");
+        PetCertificate pc2 = new PetCertificate("FDA");
+        assertNotEquals(pc1, pc2);
+    }
+
+    @Test
+    public void toStringTest() {
```

Comment:
I noticed that when testing `toString`, the method name is `toStringTest`. But when testing other methods, the method name does not contain the word `Test`. Perhaps, it would be better to standardise this? e.g removing Test from toStringTest?

Reply:
I have made the necessary amendments